composer/trainer/trainer.py

```diff
@@ -87,9 +86,6 @@ class Trainer:
             (default: ``5.0``)
         seed (int, optional): The seed used in randomization. When not provided a random seed
             will be created. (default: ``None``)
-        deterministic_mode (bool, optional): Run the model deterministically. Experimental. Performance
-            degradations expected. Certain Torch modules may not have deterministic implementations,
-            which will result in a crash. (default: ``False``)
```

Comment:
I don't believe we should remove it from calling the trainer directly as well.  Ideally we can have it both in the launcher script and the trainer creation args

Reply:
Figured out that we needed to configure it before the first test, not before the process. So resolved :)